src/main/java/it/eng/idsa/businesslogic/listener/LoggingAuditEventListener.java

```diff
@@ -25,85 +25,83 @@
 @Component
 public class LoggingAuditEventListener {
 	private static final Logger LOGGER = LoggerFactory.getLogger("JSON");
-	
+
 	private EventTypeHandler eventTypeHandler;
-	
-	private boolean isConnectorReceiver;
-	
-	public LoggingAuditEventListener(EventTypeHandler eventTypeHandler, @Value("${application.isReceiver}") boolean isConnectorReceiver) {
+
+	public LoggingAuditEventListener(EventTypeHandler eventTypeHandler,
+			@Value("${application.isReceiver}") boolean isConnectorReceiver) {
```

Comment:
Is this variable used anywhere in this class?

Reply:
It's not, I forgot to delete it from the method signature.